examples/tracing.rs

```diff
@@ -0,0 +1,24 @@
+use clap::Parser;
```

Comment:
Side note: In the future, try to do renames in separate commits so other changes are easier to review